DragonLoopAPI/Controllers/BusController.cs

```diff
@@ -14,11 +14,13 @@ public class BusController : ControllerBase
     {
         private readonly DragonLoopContext _context;
         private readonly BusManager _busManager;
+        private readonly BusHistoryManager _busHistoryManager;

```

Comment:
I'm fine having the addHistory() method be in the BusManager even though it's writing to a different table. It doesn't feel necessary to have another manager for such limited functionality.